firmware/hmi/main/peripherals/sd.h

```diff
@@ -0,0 +1,14 @@
+#ifndef __PERIPHERALS_SD_H__
+#define __PERIPHERALS_SD_H__
+
+/** General Config */
+
+/** Handlers */
+
+
+/** Prototypes */
+
+void lcd_init(void);
```

Comment:
sd.h with lcd_init function

Reply:
It's the basic example to use in future integraation, right now the code should have the core functionality but not all integrations like stream and wifi, it will be for the last step. Removed the copy and pasted lcd_init